.env

```diff
@@ -6,12 +6,12 @@ NEXTCLOUD_ADMIN_USER=admin
 NEXTCLOUD_ADMIN_PASSWORD=admin
 
 RUN_CONFIGURATION=True
-INSTALL_PLUGINS="sociallogin groupfolders files_antivirus theming_customcss"
+INSTALL_PLUGINS="groupfolders files_antivirus theming_customcss richdocuments"
 EXTERNAL_GIT_PLUGINS="https://github.com/hpi-schul-cloud/NextcloudExtract.git:extract:master https://github.com/hpi-schul-cloud/schulcloud-nextcloud-app.git:schulcloud:main"
 EXTERNAL_ARCHIVE_PLUGINS="https://github.com/hpi-schul-cloud/user_oidc/releases/download/v1.3.0-alpha/user_oidc-v1.3.0-alpha.tar.gz:user_oidc"
 ENABLE_PLUGINS="schulcloud extract theming user_oidc"
 DISABLE_PLUGINS="accessibility activity circles comments contactsinteraction dashboard files_sharing firstrunwizard password_policy photos privacy nextcloud_announcements recommendations sharebymail sociallogin support survey_client systemtags notifications updatenotification user_status weather_status"
-NEXTCLOUD_TRUSTED_DOMAINS=nextcloud.localhost nginx localhost:3100 hydra.localhost:9000
+NEXTCLOUD_TRUSTED_DOMAINS=nextcloud.localhost nginx localhost:3100 hydra.localhost:9000 collabora.localhost
```

Comment:
Maybe we need to add this also as env in helm chart. Can try without this @aimee-889